static/js/asideShowTimes.js

```diff
@@ -38,28 +37,28 @@ function sortByTime (database) {
                 days[i].push(data);
             }
         }
-    })    
+    }) 
 
     // Removes the ":" from time so it can be sorted in the next step
-    days.forEach(day => {
-        day.forEach(showtime => {
-            showtime.time = showtime.time.slice(0, 2) + showtime.time.slice(3, 5);    
-        })
-    })
+    // days.forEach(day => {
+    //     day.forEach(showtime => {
+    //         showtime.time = showtime.time.slice(0, 2) + showtime.time.slice(3, 5);    
+    //     })
+    // })
     
-    // Sorts on time
-    days.forEach(day => {
-        day.sort((a, b) => {return a.time - b.time});
-    })
+    // // Sorts on time
+    // days.forEach(day => {
+    //     day.sort((a, b) => {return a.time - b.time});
+    // })
     
-    // Adds the ":" back in time and converts dates to readable format
-    days.forEach(date => {
-        date.forEach(showtime => {
-            showtime.time = showtime.time.slice(0, 2) + ":" + showtime.time.slice(2, 4);
-            let day = new Date(showtime.day);
-            showtime.day = day.toDateString();
-        })
-    })
+    // // Adds the ":" back in time and converts dates to readable format
+    // days.forEach(date => {
+    //     date.forEach(showtime => {
+    //         showtime.time = showtime.time.slice(0, 2) + ":" + showtime.time.slice(2, 4);
+    //         let day = new Date(showtime.day);
+    //         showtime.day = day.toDateString();
+    //     })
+    // })
```

Comment:
I always recommend deleting unused code. If you want to find it again, it will still be available in the git history.